### PR TITLE
Sessions updates

### PR DESCRIPTION
**Pull request by beckykd**
### Summary
Refreshing sessions information

### Details and comments
Fixes #

### CODE REVIEW

File: docs/how_to/run_session.rst [hunk 1]
```diff
@@ -1,57 +1,137 @@
...
+
+.. _close session:
+  
+Close a session
+---------------
+
+When jobs are all done, it is recommended that you use `session.close()` to close the session. This allows the scheduler to run the next job without waiting for the session timeout,  therefore making it easier for everyone.  You cannot submit jobs to a closed session.  
+
+.. warning::  
+  Close a session only after all session jobs **complete**, rather than immediately after they have all been submitted. Jobs that are not yet queued are converted to fair-share and will likely time out.  Jobs that are queued but not completed will fail.
```
Comment (jyu00) [inline docs/how_to/run_session.rst:94]: > Jobs that are not yet queued are converted to fair-share and will likely time out.

Jobs submitted to a closed session will be rejected.

File: docs/how_to/run_session.rst [hunk 2]
```diff
@@ -1,57 +1,137 @@
...
+  ... 
+  estimator = Estimator()
+  job = estimator.run(...)
+  # Do not close here, the job might not be completed!
+  result = job.result()
+  # Reaching this line means that the job is finished.
+  session.close()
+
+Retrieve job results
+--------------------
```
Comment (jyu00) [inline docs/how_to/run_session.rst:108]: This section doesn't really belong to "Run a primitive in a session". It should be merged into the "Retrieve job results" section.

File: docs/sessions.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Benefits of using sessions
+---------------------------
+
+There are several benefits to using sessions:
+
+* Jobs that belong to a single algorithm run are run together without interruptions, increasing efficiency if your program submits multiple sequential jobs. 
+
+   .. note:: 
+    * The queuing time does not decrease for a single job submitted within a session.
```
[the remark this answers is not in the document]
Comment (beckykd) [reply docs/sessions.rst:23]: Thanks - that one felt weird to me.

File: docs/how_to/run_session.rst (hunk 1 above)
Comment (jyu00) [reply docs/how_to/run_session.rst]: Yeah, rejecting means your request doesn't even go through (i.e. you don't get a job back), and failing means your request is accepted, you get a job back, but then it fails. What you have now is fine.

**Review by jyu00 — APPROVED** [review]

**Pull request merged**